atlp-rwanda/phantom-fe-elite
# #23 Feature: Crud operation bus integration

## What does this PR do?
Create, read, update and delete bus.
## Description of Task to be completed?
As an operator, I want to create buses so that driver can drive the pre-defined buses.

- The operator should be able to create, delete, and update a bus.
- The operator should be able to view a paginated list of buses in the system.
- Appropriate error messages are displayed on unsuccessful attempts
- ***The integration of its API from backend should be working***

## How should this be manually tested?
- `Clone the repo`
- `checkout` to branch `ft-crud-operation-bus-integration-23
- pull the changes by `git pull`
- install all packages by running `npm install`
- start app by `running npm start`

***As of note***: Before running the app, open the backend and start the server as by now it is connected to the local backend service. for signing in. Check the *credentials* on the user seed file in the backend repo to view them, including ***email** and **pasword** to use*

## Any background context you want to provide?
When deployed version of backend service is done, `I will replace the local path to the deployed path of API`.
## What are the relevant pivotal tracker stories?
[Trello Board](https://trello.com/c/Bgd5JZZR/23-crud-operations-for-buses)

Screenshots (if appropriate)
N/A
